tests/Unit/Domain/Test_DomainCreators.cpp

```diff
@@ -224,3 +294,118 @@ SPECTRE_TEST_CASE("Unit.Domain.DomainCreators.TestHelperFunctions",
+           {Direction<3>::lower_eta(), {3, quarter_turn_cw_about_zeta}},
+           {Direction<3>::lower_zeta(), {6, aligned_orientation}}},
+          {{Direction<3>::upper_xi(), {4, quarter_turn_cw_about_zeta}},
+           {Direction<3>::upper_eta(), {3, quarter_turn_cw_about_zeta}},
+           {Direction<3>::lower_xi(), {5, quarter_turn_ccw_about_zeta}},
+           {Direction<3>::lower_eta(), {2, quarter_turn_cw_about_zeta}},
+           {Direction<3>::lower_zeta(), {6, center_relative_to_minus_z}}},
+          {{Direction<3>::upper_xi(), {0, quarter_turn_ccw_about_zeta}},
+           {Direction<3>::upper_eta(), {4, quarter_turn_cw_about_zeta}},
+           {Direction<3>::lower_xi(), {1, quarter_turn_ccw_about_zeta}},
+           {Direction<3>::lower_eta(), {5, quarter_turn_cw_about_zeta}},
+           {Direction<3>::lower_zeta(), {6, center_relative_to_plus_y}}},
+          {{Direction<3>::upper_xi(), {1, quarter_turn_ccw_about_zeta}},
+           {Direction<3>::upper_eta(), {4, quarter_turn_ccw_about_zeta}},
+           {Direction<3>::lower_xi(), {0, quarter_turn_ccw_about_zeta}},
+           {Direction<3>::lower_eta(), {5, quarter_turn_ccw_about_zeta}},
+           {Direction<3>::lower_zeta(), {6, center_relative_to_minus_y}}},
+          {{Direction<3>::upper_xi(), {2, quarter_turn_ccw_about_zeta}},
+           {Direction<3>::upper_eta(), {0, quarter_turn_cw_about_zeta}},
+           {Direction<3>::lower_xi(), {3, quarter_turn_cw_about_zeta}},
+           {Direction<3>::lower_eta(), {1, quarter_turn_ccw_about_zeta}},
+           {Direction<3>::lower_zeta(), {6, center_relative_to_plus_x}}},
+          {{Direction<3>::upper_xi(), {3, quarter_turn_cw_about_zeta}},
+           {Direction<3>::upper_eta(), {0, quarter_turn_ccw_about_zeta}},
+           {Direction<3>::lower_xi(), {2, quarter_turn_ccw_about_zeta}},
+           {Direction<3>::lower_eta(), {1, quarter_turn_cw_about_zeta}},
+           {Direction<3>::lower_zeta(), {6, center_relative_to_minus_x}}},
+          {{Direction<3>::upper_zeta(), {0, aligned_orientation}},
+           {Direction<3>::lower_zeta(),
+            {1, center_relative_to_minus_z.inverse()}},
+           {Direction<3>::upper_eta(),
+            {2, center_relative_to_plus_y.inverse()}},
+           {Direction<3>::lower_eta(),
+            {3, center_relative_to_minus_y.inverse()}},
+           {Direction<3>::upper_xi(), {4, center_relative_to_plus_x.inverse()}},
+           {Direction<3>::lower_xi(),
+            {5, center_relative_to_minus_x.inverse()}}}},
+
+      std::vector<std::unordered_set<Direction<3>>>{
+          {{Direction<3>::upper_zeta()}},
+          {{Direction<3>::upper_zeta()}},
+          {{Direction<3>::upper_zeta()}},
+          {{Direction<3>::upper_zeta()}},
+          {{Direction<3>::upper_zeta()}},
+          {{Direction<3>::upper_zeta()}},
+          {}});
+}
+
+SPECTRE_TEST_CASE("Unit.Domain.DomainCreators.Sphere.Factory",
```

Comment:
same comments as the disk test, split into separate file, and test the sphere created via the factory